FWCore/Framework/src/DataKey.cc

```diff
@@ -22,132 +22,100 @@
 // constants, enums and typedefs
 //
 
-static const char kBlank[] = {'\0'};
```

Comment:
Please return the spacing back to what it was for this file.

Reply:
Will do.